### PR TITLE
SPOI-6758 #comment added trouble shouting for kryo

### PR DESCRIPTION
**Pull request by gauravgopi123**

### CODE REVIEW

**Comment by gauravgopi123** [comment]
@tweise Please review it

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -324,6 +324,50 @@ Add the property :
 
 Check if the host where gateway is running has yarn-site.xml file. You need to have all Hadoop configuration files accessible to dtGateway for it to run successfully.
 
+### Application throws following Kryo exception.
+
+      com.esotericsoftware.kryo.KryoException: Class cannot be created (missing no-arg constructor): 
+
+This means that kryo is not able to deserialize the object because the type is missing default constructor. There are couple of ways to address this exception
```
Comment (tweise) [inline docs/troubleshooting.md]: Kryo

File: docs/troubleshooting.md [hunk 2]
```diff
@@ -324,6 +324,50 @@ Add the property :
...
+
+1. Add default constructor to the type in question. 
+2. Using existing alternative serializer for the type in question. Custom serializer can be used as following as follows
```
Comment (tweise) [inline docs/troubleshooting.md]: Perhaps link to Kryo, also refer to existing alternate serializer? Most of the time user does not need to write one.

File: docs/troubleshooting.md [hunk 3]
```diff
@@ -324,6 +324,50 @@ Add the property :
...
+
+    2.1 Using Kryo's @FieldSerializer.Bind annotation for the field throwing exception. Here is how to bind custom serializer.
+    ```java
+    @FieldSerializer.Bind(CustomSerializer.class)
+    SomeType someType
+    ```
+    Kryo will use this CustomSerializer to serialize and deserialize type SomeType.
+
+    2.2 Using custom serializer with stream codec. You need to define custom stream codec and attach this custome codec to the input port that is expecting the type in question. Following is an example of creating custom Stream Codec.
```
Comment (tweise) [inline docs/troubleshooting.md]: Following ... Stream Codec:

**Comment by gauravgopi123** [comment]
Incorporated feedback

**Pull request merged**